files/image_config/monit/monitrc

```diff
@@ -16,7 +16,7 @@
 ##
 ## Start Monit in the background (run as a daemon):
 #
-  set daemon 120            # check services at 2-minute intervals
+  set daemon 60            # check services at 1-minute intervals
```

Comment:
Add extra space before comment to fix alignment

Reply:
Ok, I will change all the existing SONiC monit configurations and add an extra space to fix alignment.